src/MOLFiniteDifference/MOL_discretization.jl

```diff
@@ -118,12 +122,15 @@ function SciMLBase.symbolic_discretize(pdesys::ModelingToolkit.PDESystem,discret
         # TODO: Generalize central difference handling to allow higher even order derivatives
         # The central neighbour indices should add the stencil to II, unless II is too close
         # to an edge in which case we need to shift away from the edge
+
         # Calculate buffers
-        I1 = oneunit(first(indices))
-        Imin = first(indices) + I1 * (order÷2)
-        Imax = last(indices) - I1 * (order÷2)
+        # TODO: check central_neighbor_idxs. It does not work like the old implementation.
+        #I1 = oneunit(first(indices))
+        #Imin = first(indices) + I1 * (order÷2)
+        #Imax = last(indices) - I1 * (order÷2)
         # Use max and min to apply buffers
-        central_neighbor_idxs(II,j) = stencil(j) .+ max(Imin,min(II,Imax))
+        # central_neighbor_idxs(II,j) = stencil(j) .+ max(Imin,min(II,Imax))  
```

Comment:
? just remove if there's a new definition

Reply:
it's back to the old definition which ignores the discretization order (I think) i.e. breaks #365 but not enough to cause tests to fail

Reply:
weird.

Reply:
Are you looking into this?

Reply:
I can take a look tomorrow

Reply:
The only thing that is broken with the `stencil` approach is "Test 00: Dt(u(t,x,y)) ~ Dxx(u(t,x,y)) + Dyy(u(t,x,y))" in the `MOL_2D_Diffusion.jl` test with orders 4 and 6. The test breaks because the solution is not accurate enough, not a syntax error. The reason it works with the old syntax is that it just ignores `discretization.centered_order` and pretends you're always asking for order 2. @emmanuellujan if you only test for order 2 in that test then you can use the syntax you commented out, and we should open an issue for the order 4 and 6 in 2D.

Reply:
I understand that it is better not to remove those comments for now.

Reply:
actually I'll just open a PR to your fork

Reply:
Oh, we posted almost at the same time, I did not see your comment when I posted mine. I will check what you mention. Thank you :)